package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "kf-portal-ui",
-  "version": "3.1.3",
+  "version": "3.2.0",
```

Comment:
I would bump version in a separate commit which  would include bump + release note
The release tag will be on this specific commit 
Just to clarify history :)